src/ideations/ideations.controller.ts

```diff
@@ -102,10 +178,35 @@ export class IdeationsController {
         summary:
             "Deletes an ideation given a ideationId (int) and that the user that created it is logged in.",
     })
+    @ApiResponse({
+        status: HttpStatus.UNAUTHORIZED,
+        description:
+            "Invalid uuid or teamID. User is not authorized to preform this action.",
+        type: UnauthorizedErrorResponse,
+    })
+    @ApiResponse({
+        status: HttpStatus.NOT_FOUND,
+        description: "Ideation with given ID does not exist.",
+        type: NotFoundErrorResponse,
+    })
+    @ApiResponse({
```

Comment:
Can you check this one? I think the type of response used here should an error type instead of IdeationResponse.